src/main/java/de/tum/www1/orion/ui/assessment/OrionTemplateDiffEditorProvider.kt

```diff
@@ -1,3 +1,4 @@
+@file:Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
```

Comment:
That is a quite interesting behavior. I would guess it only occurs since the newest version of JetBrains Api

Reply:
Yes. They changed it to internal with the commit https://github.com/JetBrains/intellij-community/commit/16c9cd96c76cdeba729f3f6c3dd1ed7adab2a95c

Reply:
> The proposed changes look good to me. The OrionTemplateDiffEditorProvidor change might need refactoring in the future but the code works well for now.
Yes. Correct. That's the next step.